node-js-basics-101/node_modules/after/test/after-test.js

```diff
@@ -0,0 +1,120 @@
+/*global suite, test*/
+
+var assert = require("assert")
+    , after = require("../")
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.

<!-- Comment by [SideCI](https://sideci.com) -->